### PR TITLE
DocBook validation fixes (part 1)

### PR DESCRIPTION
**Pull request by dtardon**
This should be uncontroversial stuff--either obvious fixes or trivial changes.

### CODE REVIEW

File: man/cgroup-sandboxing.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
 -->
 
 <refsect1>
+  <title/>
 
 <para id="singular">This option cannot be bypassed by prefixing <literal>+</literal> to the executable path
```
Comment (yuwata) [inline man/cgroup-sandboxing.xml]: Could you adjust indentation here and below?

File: man/supported-controllers.xml [hunk 2]
```diff
@@ -6,6 +6,7 @@
 -->
 
 <refsect1>
+  <title/>
 
 <para id="controllers-text">The following controller names may be specified: <option>cpu</option>, <option>cpuacct</option>,
```
Comment (yuwata) [inline man/supported-controllers.xml]: Also here.

File: man/threads-aware.xml [hunk 3]
```diff
@@ -4,6 +4,7 @@
 <!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
 
 <refsect1>
+  <title/>
 
 <para id="strict">All functions listed here are thread-agnostic and only a single specific thread may operate on a
```
Comment (yuwata) [inline man/threads-aware.xml]: Also here.

**Review by yuwata — APPROVED** [review]
LGTM. Several minor comments.

File: man/cgroup-sandboxing.xml (hunk 1 above)
Comment (dtardon) [reply man/cgroup-sandboxing.xml]: Sure.

**Review by yuwata — APPROVED** [review]
Thank you.

**Pull request merged**